### ISSUE TITLE
Probably missing localization for the Grid View in Drives

### ISSUE THREAD

**Issue by fabriziocacicia** [guydavis/machinaris#630]
I was updating the Italian translations for the **Drives** page and I noticed that a string from the Grid View was missing, in particular: "Powered On".
So, I went into the HTML source of the page and I noticed that none of the strings for the Grid View have the localization applied. Am I wrong?

```html
            <table class="text-muted" style="width: 100%">
              <tr>
                <td>S.M.A.R.T</td>
                <td>Temperature</td>
                <td>Capacity</td>
                <td>Powered On</td>
              </tr>
              <tr>
```

I think it should be something like:
```html
            <table class="text-muted" style="width: 100%">
              <tr>
                <td>S.M.A.R.T</td>
                <td>{{_('Temperature')}}</td>
                <td>{{_('Capacity')}}</td>
                <td>{{_('Powered On')}}</td>
              </tr>
              <tr>
```

**Comment by guydavis** [comment]
Good catch!  Now fixed in the `develop` branch.    Please create a MR against that branch for next release.  Thanks again.

**Comment by guydavis** [comment]
This fix is now available in the [Machinaris v0.7.2 release](https://github.com/guydavis/machinaris/releases/tag/0.7.2) today. Thanks!
